RELEASE-NOTES.txt

```diff
@@ -6,6 +10,7 @@
 * [*] Fixed an issue where the push notifications prompt button would overlap on iPad. [#19304]
 * [*] Story Post: Fixed an issue where deleting one image in a story draft would cause the following image not to load. [#16966]
 * [*] Fixed an issue where the no result label on the side menu is oversize on iPad. [#19305]
+* [*] [internal] Various low level database refactors around posts, pages, and comments. [#19353, #19363, #19386]
```

Comment:
cc @crazytonyli, I decided to add this line so that it can be tracked as part of the things to be particularly mindful of (and maybe to try and break) during beta testing.

Reply:
Oh nice! Thanks for that. I'll remember to do that next time.

Reply:
👍 Yeah, no worries. It's tricky because it doesn't make sense to spam the release notes with entry for ever granular refactor PR but it's still handy to let people know these things have happened from a high level point-of-view.